OrbitVulkanLayer/PhysicalDeviceManager.cpp

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
Why the empty file? Doesn't it just cause a bit of confusion? In particular since the class is templated there will probably be no methods here. Also it's not even added to CMakeLists.txt.

Reply:
I deleted it. Sorry for the noise.